### ISSUE TITLE
number of GPUS

### ISSUE THREAD

**Issue by ztt0821** [JialeCao001/SipMask#35]
May I ask how many gpu cards you have used to train sipmask-mmdetection. Because I found that in mmdetection, we need to change the learning rate if number of GPU cards are different

**Comment by JialeCao001** [comment]
@ztt0821 Thanks. 4 gpus. 4 images per gpu.

**Comment by ztt0821** [comment]
> @ztt0821 Thanks. 4 gpus. 4 images per gpu.

Thank you for your answer. May I ask the meaning of 6x and 10x in SipMask-mmdetection? I found the epochs are all 24.

**Comment by ztt0821** [comment]
Thank you very much
